hogben/utils.py

```diff
@@ -200,8 +200,10 @@ def fisher(qs, xi, counts, models, step=0.005):
             lb = np.array([param.bounds.limits[0] for param in xi])
             ub = np.array([param.bounds.limits[1] for param in xi])
 
+        if importance is None: # Default importance equals one for each param
+            importance = np.ones(len(lb))
         # Using the equations from the paper for the coordinate transform.
-        H = np.diag(1/(ub-lb))
+        H = np.diag(importance/(ub-lb))
```

Comment:
I think this might result in the importance being squared

Reply:
That is correct, as the J matrix (in article) gets multiplied on both sides, in the current implementation (latest commit), it scales to the importance directly. Corresponding to a scaling with varience.